.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Docs to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: yarn
+
+      - name: Change to docs directory
+        run: cd docs
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build the Docusaurus site
+        run: yarn build
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          deploy_key: ${{ secrets.DOCS_DEPLOY_KEY }}
+          external_repository: 'bagetter/bagetter.github.io'
+          publish_dir: ./docs/build
```

Comment:
In my tests I needed to change the line to the following to only have the content of `build` in the external repo: `publish_dir: ./build`

Reply:
Probably searches from the set working-directory.